### PR TITLE
Configure communication service connections

### PR DESCRIPTION
**Pull request by weshaggard**

### CODE REVIEW

File: sdk/communication/tests.yml [hunk 1]
```diff
@@ -25,11 +25,12 @@ extends:
       ServiceDirectory: communication
```
Comment (weshaggard) [inline sdk/communication/tests.yml:25]: Should this test pipeline even exist given it is split out by package already?

**Review by benbp — APPROVED** [review]

File: sdk/communication/tests.yml (hunk 1 above)
Comment (benbp) [reply sdk/communication/tests.yml:25]: Seems like a no to me

**Comment by weshaggard** [comment]
While I can make this update the current subscription configured for communication is at quote and cannot be deployed to so this will not work. I think we will need some owners from the communications team to help move these tests pipelines forward.

**Review by ilyapaliakou-msft — APPROVED** [review]

**Comment by weshaggard** [comment]
For now until we get more assistance from the communication team to clean their subscriptions I'm going to keep these using our public TME sub.

**Pull request closed without merging**